### DISCUSSION TITLE
Can not login to GitHub with the copilot plug in

### DISCUSSION THREAD

**Discussion by Omerdan03** [community/community#42289] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

![image](https://user-images.githubusercontent.com/57575138/208897413-068b356f-3500-44de-b90c-e86dd980dbaa.png)

I'm clicking the login to GitHub some window pop up and it can't connet.
Any ideas what can I try next?

This is happening in both pycharm and intellij

EDIT:
had to close the reopen both IDEs and it worked

**Comment by iamcjwilliams** [comment]
@Omerdan03 Please visit here to troubleshoot your issue: https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-github-copilot-in-a-jetbrains-ide
